pkg/cli/kubernetes/logstream/stern.go

```diff
@@ -81,7 +81,7 @@ func (i *Impl) Stream(ctx context.Context, options Options) error {
 		ErrOut:     options.Out,
 
 		// Limit the number of concurrent log fetch function unlike Kubernetes client rate limitter.
-		MaxLogRequests: 10,
+		MaxLogRequests: 50,
```

Comment:
Should we update this comment with some context on user experience if an app exceeds this limit, and that this is randomly selected number?

Reply:
I will create a follow-up PR to update the comment here. (Apologies for missing this comment.)